docs/guide/adapters/emberznet.md

```diff
@@ -0,0 +1,190 @@
+* [Flashing](https://sonoff.tech/wp-content/uploads/2022/08/SONOFF-Zigbee-3.0-USB-dongle-plus-firmware-flashing-.pdf)
+* [Buy](https://itead.cc/product/zigbee-3-0-usb-dongle/)
+
+<img src="../../images/dongle-e.jpg" width="200" />
+</details>
+
+<details>
+<summary>Nabu Casa Home Assistant SkyConnect (EFR32MG21)</summary>
+
+With integrated antenna and hardware flow control support.
+
+```yaml
+serial:
+  rtscts: true
+```
+
+* [Product page](https://www.home-assistant.io/skyconnect)
+* [Coordinator firmware](https://github.com/NabuCasa/silabs-firmware)
+* [Flashing](https://skyconnect.home-assistant.io/)
+* [Buy](https://www.home-assistant.io/skyconnect)
+
+<img src="../../images/skyconnect_isometric.jpg" width="200" />
+</details>
+
+<details>
+<summary>SMLIGHT SLZB-07 (EFR32MG21)</summary>
+
+With external antenna and hardware flow control support.
+
+```yaml
+serial:
+  rtscts: true
+```
+
+* [Product page](https://smlight.tech/product/slzb-07/)
+* Buy: [Official Store - Worldwide](https://smartlight.me/smart-home-devices/zigbee-devices/slzb-07-zigbee-usb-adapter-en), [Aliexpress - Worldwide](https://www.aliexpress.com/item/1005006273914143.html).  
+Local fast delivery: [Australia](https://shop.dialedin.com.au/products/smlight-slzb-07-usb-zigbee-adapter), [Austria, Belgium, Germany, Netherlands](https://www.hobbyelectronica.nl/product/slzb-07-zigbee-coordinator-usb/), [France](https://www.domadoo.fr/fr/interface-domotique/7044-smlight-dongle-usb-zigbee-efr32mg21-cp2102n-soc-antenne-3db-zigbee2mqtt-et-zha.html), [Poland](https://pcblab.io/szukaj?controller=search&s=slzb), [US Amazon](https://www.amazon.com/LOAMLIN-SMLIGHT-SLZB-07-Zigbee-Coordinator/dp/B0CNVBCCR3)
+
+<img src="../../images/smlight-slzb-07.jpg" width="200" />
+
+</details>
+
+### Network (TCP)
+
+
+<details>
+<summary>TubeZB EFR32 MGM24 POE (MGM240PB32VNN)</summary>
+
+With external antenna.
+
+* [Product page](https://tubeszb.com/product/efr32-mgm24-poe-coordinator/)
+* [Buy](https://tubeszb.com/product/efr32-mgm24-poe-coordinator/)
+
+<img src="../../images/tubezb-mgm24-poe.jpg" width="200" />
+
+</details>
+
+
+### Hybrid (USB + Network)
+
+<details>
+<summary>SMLIGHT SLZB-06mg24 (EFR32MG24)</summary>
+
+With external antenna.
+
+* [Product page](https://smlight.tech/product/slzb-06mg24/)
+* [Buy] UPCOMING
+
+<img src="../../images/smlight-slzb-06mg24.jpg" width="200" />
+
+</details>
+
+
+### Hub
+
+<details>
+<summary>Nabu Casa Home Assistant Yellow (MGM210P)</summary>
+
+With integrated antenna and hardware flow control support.
+
+```yaml
+serial:
+  rtscts: true
+```
+
+* [Product page](https://www.home-assistant.io/yellow)
+* [Coordinator firmware](https://github.com/NabuCasa/silabs-firmware)
+* [Flashing](https://skyconnect.home-assistant.io/)
+* [Buy](https://www.home-assistant.io/yellow)
+
+<img src="../../images/yellow.jpg" width="200" />
+</details>
+
+### Custom
+
+
+## Not recommended
+
+### Known issues
+
+- SMLight SLZB-06M
```

Comment:
What are the known issues with this one?

Reply:
It had firmware issues. And it seems to still have some. darkxst is investigating. Since SMLight has the newer SLZB-06mg24, better to put this one in recommended I think?

Reply:
Yes definitely, but please put a link to the github issue to explain what issues it has.